lib/src/module.dart

```diff
@@ -142,6 +142,8 @@ abstract class Module {
     } else if (reserveDefinitionName &&
         !Sanitizer.isSanitary(definitionName!)) {
       throw InvalidReservedNameException();
+    } else if (definitionName == '') {
+      throw NullReservedNameException();
```

Comment:
Good catch on this same bug for `definitionName` in module, but what about `name`?

Reply:
Why resolved?

Reply:
My bad, I thought name and definition share on `_nameValidation` function. I added `_nameValidation` to the name as well. It should have the same behaviour now.